src/dune/__main__.py

```diff
@@ -220,8 +240,21 @@ def load_all_modules_from_dir(plugin_dir):
             elif args.bootstrap_system:
                 dune_sys.bootstrap_system(False)
 
-            elif args.bootstrap_system_full:
-                dune_sys.bootstrap_system(True)
+            elif args.bootstrap_system_full is not None:
+                if len(args.bootstrap_system_full) > 2:
+                    dune_sys.bootstrap_system(True,
+                                            args.bootstrap_system_full[0],
+                                            args.bootstrap_system_full[1],
+                                            args.bootstrap_system_full[2])
+                elif len(args.bootstrap_system_full) > 1:
+                    dune_sys.bootstrap_system(True,
+                                            args.bootstrap_system_full[0],
+                                            args.bootstrap_system_full[1])
+                elif len(args.bootstrap_system_full) > 0:
+                    dune_sys.bootstrap_system(True,
+                                            args.bootstrap_system_full[0])
+                else:
+                    dune_sys.bootstrap_system(True)
```

Comment:
While python isn't my favorite language, it's still pretty cool. I think we can make this more robust and simpler with a small change. Check it out and tell me what you think.
```suggestion
                if len(args.bootstrap_system_full) > 3:
                    # Some error state belongs here.                                                                                             
                else:
                    dune_sys.bootstrap_system(True, *args.bootstrap_system_full);

```

Reply:
Great trick! Thanks